deployment/requirements.txt

```diff
@@ -0,0 +1,24 @@
+ansible==2.1.1.0
```

Comment:
This seems like it may have been extracted from the output of `pip freeze`. In general, I think we'd want to only include top level dependencies inside of `requirements.txt` and defer the rest of the work to the dependency constraints of the top level dependencies themselves.